packages/broker/src/plugins/metrics/MetricsPlugin.ts

```diff
@@ -11,8 +11,10 @@ const logger = new Logger(module)
 
 export interface MetricsPluginConfig {
     consoleAndPM2IntervalInSeconds: number
+    firehoseStreamIdHead: string
```

Comment:
Is this one used (looks like the one inside `nodeMetrics` section is only used?), can be removed?

Reply:
good catch